src/IconMenu/IconMenu.js

```diff
@@ -18,6 +18,11 @@ class IconMenu extends Component {
      */
     anchorOrigin: propTypes.origin,
     /**
+     * If true, the popover will apply transitions when
+     * added it gets added to the DOM.
```

Comment:
This was in the original file.
It was not indent therefor the previous comment (also was in the original file).
I fixed it anyhow.
I will also try to understand what the intention here and fix this as well.

This feature is important to us and it would be appreciated if you could  promote it